src/app/routes/homePage/getInitialData/index.ts

```diff
@@ -0,0 +1,79 @@
+/* eslint-disable @typescript-eslint/ban-ts-comment */
+import { Agent } from 'https';
+import Url from 'url-parse';
+import nodeLogger from '../../../lib/logger.node';
+import { BFF_FETCH_ERROR } from '../../../lib/logger.const';
+import fetchPageData from '../../utils/fetchPageData';
+import { Services } from '../../../models/types/global';
+import HOME_PAGE_CONFIG from './page-config';
+
+const logger = nodeLogger(__filename);
+
+type Envs = 'test' | 'live' | 'local' | 'undefined';
+
+const getEnvironment = (pathname: string) => {
+  if (pathname.includes('renderer_env=test')) {
+    return 'test';
+  }
+  if (pathname.includes('renderer_env=live')) {
+    return 'live';
+  }
+
+  return process.env.SIMORGH_APP_ENV as Envs;
+};
+
+type Props = {
+  getAgent: () => Promise<Agent>;
+  service: Services;
+  path: string;
+  pageType: 'homePage';
+};
+
+export default async ({ getAgent, service, path: pathname }: Props) => {
+  try {
+    const env = getEnvironment(pathname);
+    const isLocal = !env || env === 'local';
+
+    const agent = !isLocal ? await getAgent() : null;
+    const id = !isLocal ? HOME_PAGE_CONFIG[service][env] : null;
+
+    let fetchUrl = Url(process.env.BFF_PATH as string).set('query', {
+      id,
+      service,
+      pageType: 'home',
```

Comment:
What if we used the pageType which is available as a parameter here? And update https://github.com/bbc/simorgh/blob/61a36b98bb651fd629bce75ffd3b10039558913c/src/app/routes/utils/pageTypes.js#L16 to be lowercase: `'home'`  

**Future:** We might also want to update the simorgh-bff, to allow pageType to be case-insensitive